lib/dry/view/exposures.rb

```diff
@@ -40,6 +40,14 @@ def locals(input)
         }
       end
 
+      def each(&block)
+        exposures.each(&block)
+      end
+
+      def import(name, exposure)
```

Comment:
Would you mind moving this method underneath `#add`? Feels like the right ordering to me.